core/src/main/scala/kafka/server/ConfigHandler.scala

```diff
@@ -201,6 +201,18 @@ class BrokerConfigHandler(private val brokerConfig: KafkaConfig,
       quotaManagers.follower.updateQuota(upperBound(getOrDefault(FollowerReplicationThrottledRateProp)))
       quotaManagers.alterLogDirs.updateQuota(upperBound(getOrDefault(ReplicaAlterLogDirsIoMaxBytesPerSecondProp)))
     }
+
+    def updateBrokerThrottle(prop: String, quotaManager: ReplicationQuotaManager): Unit = {
+      if (properties.containsKey(prop) && properties.getProperty(prop).equals("true")) {
+        debug(s"Removing $prop on broker ${brokerConfig.brokerId}")
```

Comment:
I vote we put this in `info()`. I think this type of config will be changed pretty rarely so it's useful to have an explicit log to note the change

Reply:
Curious to see what other thing about logging these replication throttle changes. Another one I would do is add a log in whenever the `.rate` is changed